exercises/spiral-matrix/description.md

```diff
@@ -0,0 +1,22 @@
+Given a number, return a matrix of numbers in spiral order.
+
+The matrix should be filled with natural numbers, starting from 1
+in the top-left corner, increasing in a inward, clockwise spiral order,
+like these examples:
+
+- Spiral matrix of size 2
+
+    ```plain
```

Comment:
`Spiral matrix of size 2` - where does the 2 come from here? The given matrix is 3x3!